examples/ogbn_train.py

```diff
@@ -0,0 +1,446 @@
+# Reaches around 0.7870 ± 0.0036 test accuracy.
+
+import argparse
+import os.path as osp
+import time
+from typing import Tuple
+
+import torch
+import torch.nn.functional as F
+from ogb.nodeproppred import Evaluator, PygNodePropPredDataset
+from torch import Tensor
+from torch.nn import Linear
+from tqdm import tqdm
+
+from torch_geometric.loader import NeighborLoader
+from torch_geometric.nn import GATConv, SAGEConv
+from torch_geometric.utils import to_undirected
+
+parser = argparse.ArgumentParser(
+    formatter_class=argparse.ArgumentDefaultsHelpFormatter, )
+parser.add_argument(
+    '--dataset',
+    type=str,
+    default='ogbn-papers100M',
+    choices=['ogbn-papers100M', 'ogbn-products'],
+    help='Dataset name.',
+)
+parser.add_argument(
+    '--dataset_dir',
+    type=str,
+    default='./data',
+    help='Root directory of dataset.',
+)
+parser.add_argument(
+    "--dataset_subdir",
+    type=str,
+    default="ogb-papers100M",
+    help="directory of dataset.",
+)
+parser.add_argument(
+    '--use_gat',
+    action='store_true',
+    help='Whether or not to use graphsage model',
+)
+parser.add_argument(
+    '--verbose',
+    action='store_true',
+    help='Whether or not to generate statistical report',
+)
+parser.add_argument(
+    '--test_inference',
+    action='store_true',
+    help='Whether or not to test inference method',
+)
+parser.add_argument('--device', type=str, default='cuda')
+parser.add_argument('--runs', type=int, default=1, help='number of runs.')
```

Comment:
```suggestion
```

Reply:
Just run once. Since the goal of examples is to show new users how to use the library.